### DISCUSSION TITLE
Custom Nemo shortcut for move/copy to other pane

### DISCUSSION THREAD

**Discussion by morcoveata** [linuxmint/discussions#659] [Questions]
In the nemo file manager in linux mint cinnamon I'm trying to make the F5 and F6 keys to override their default behavior and be used to copy/move files from one panel to another, after pressing F3. I tried several variants, including the one on the [viewtopic.php?t=311974](https://forums.linuxmint.com/viewtopic.php?t=311974) post, but it didn't work. I have nemo 6.0.2 version.

**Comment by Jopp-gh** [comment]
Agreed. Currently, the only way to do what you want, is to right-mouse-click on files and folders, then select "move to" or "copy to" in the contextual menu.

I proposed to expand Nemo's vocabulary to include selections:
https://github.com/orgs/linuxmint/discussions/584
Adding selections would be incredible useful for scripting.

However, the team of Mint seems to be very small and their main priority right now is to consolidate Mint even further and prepare it for the future. 

On top of that, Wilma redesigns the user interface, which should be very low level priority, Mint doesn't need a redesign. (Wilma adds also too much rounded borders, which eats just space - in my opinion). Maybe this "design phase" is just a break, before the team of Mint turns their attention to more pressing issues.
